python/ql/test/library-tests/dataflow/tainttracking/commonSanitizer/test_const_compare.py

```diff
@@ -85,6 +85,27 @@ def test_in_local_variable():
     else:
         ensure_tainted(ts) # $ tainted
 
+def test_is_none():
+    ts = TAINTED_STRING
+    if ts is None:
+        ensure_not_tainted(ts)
+    else:
+        ensure_tainted(ts) # $ tainted
+
+def test_is_not_none():
+    ts = TAINTED_STRING
+    if ts is not None:
+        ensure_tainted(ts)  # $ tainted
+    else:
+        ensure_not_tainted(ts)
+
+def test_in_list_with_constants():
+    ts = TAINTED_STRING
+    if ts in ["safe", None, 3, False]:
```

Comment:
Would it be considered tainted if there is a non-constant in this list? Should we have a test with that?

Reply:
Additional test added.